general.css

```diff
@@ -0,0 +1,973 @@
+    }
+
+    .portafolio-images .team-image__container {
+        grid-row-start: span 1;
+        grid-column-start: span 1;
+    }
+
+    .blog-comment__exception{
+        display: grid;
+        grid-template-rows: 100%;
+        grid-template-columns: repeat(2,50%);
+    }
+
+    .blog-comment__exception .blog-comment{
+        grid-row-start: span 1;
+        grid-column-start: span 1;
+    }
+    
+    .blog-comment:active ,
+    .blog-comment:hover {
+        background: url("images/hover-image.jpg") no-repeat;
+        background-size: cover;
+        background-position: bottom -5rem left 0;
+    }
+    
+    .blog-comment__exception .blog-comment:active, 
+    .blog-comment__exception .blog-comment:hover{
+        background: url("images/hover-image.jpg") no-repeat;
+        background-size: auto 50vh;
+        background-position: bottom -1rem left 0;
+    }
+
+    .question{
+        width: 40rem;
+    }
+
+    .testimonial{
+        align-items: center;
+    }
+
+    .testimonial-1{
+        display: none ;
+    }
+
+    .testimonial-2{
+        display: flex;
+        flex-direction: column;
+
+    }
+    .testimony__item{
+        display: grid;
+        grid-template-rows: 100%;
+        grid-template-columns: repeat(2,50%);
+    }
+
+    .testimony{
+        display: flex;
+    }
+
+    .contact-form{
+        max-width: 35rem;
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+        align-items: center;
+        
+    }
+
+   .input-text{
+       padding-right:  1rem;
+    }
+
+   .contact-button__sub{
+       width:  20rem;
+       text-align: center;
+    }
+
+
+}
+
+
+@media (min-width:48rem){
+
+    .carrousel-images{
+        height: 100vh; 
+    }
+
+
+    .main-nav{
+        justify-content: space-between;
+        align-items: center;
+        padding: 0 3.5rem;
+    }
+
+    .nav-bar__item{
+        margin: 1rem;
+    }
+
+    .section-about{
+        flex-direction: row;
+    }
+
+    .about__video{
+        width: 150vw;
+        height:40vh;
+    }
+
+    .team{
+        display: grid;
+        grid-template-rows: 100%;
+        grid-template-columns: repeat(3,33%);
+    }
+
+    .team-member {
+        grid-column-start: span 1;
+    }
+
+    .portafolio-images{
+        display: grid;
+        grid-template-rows: repeat(2,50%);
+        grid-template-columns: repeat(3,33%);
+    }
+
+    .portafolio-images .team-image__container {
+        grid-row-start: span 1;
+        grid-column-start: span 1;
+    }
+
+    .blog-comments{
+        display: grid;
+        grid-template-rows: repeat(2,50%);
+        grid-template-columns: repeat(2,50%);
+    }
+
+    .blog-comment{
+        grid-row-start: span 1;
+        grid-column-start: span 1;
+    }
+
+    .blog-comment__exception{
+        grid-row: 2 / span 1;
+        grid-column: 1 / span 2;
+    }
+    
+    .blog-comment:active ,
+    .blog-comment:hover {
+        background: url("images/hover-image.jpg") no-repeat;
```

Comment:
this line is unnecessary because you already put this in line 835, you don't have to repeat it, just put what changes.